languages/common-lisp/exercises/concept/sexpr/package.lisp

```diff
@@ -1,3 +1,6 @@
+(push (uiop:getcwd) asdf:*central-registry*)
```

Comment:
While this works it doesn't feel right 1) because using `*central-registry*` is definitely not recommended anymore in ASDF 2) the package file is not the place for this (IMHO).

If we do go with such an approach I'd say we should create another file - maybe `boot.lisp` or `run.lisp` which would do this sort of stuff and then the  student could load that to run load files and run tests.

(Oh and another thing: `push` means you'll have lots of copies of the current directory in the registry. Probably want to use `pushnew`...)

Reply:
Yeah, it's quite hacky. I think we might be better off without ASDF (a good call on `push` vs `pushnew` though!)